src/user/entities/user.entity.ts

```diff
@@ -0,0 +1,7 @@
+export class User {
```

Comment:
may be we can use interface instead of class here

Reply:
This object is generated automatically by means of nest, I just filled in the fields. Why is an interface better than a class?